lib/blocks.php

```diff
@@ -131,6 +131,33 @@ function gutenberg_serialize_block( $block ) {
 	return $content;
 }
 
+/**
+ * Renders a single block into a HTML string.
+ *
+ * @since 1.3.0
```

Comment:
This needs to be updated.

Reply:
Fixed in b8f96e9b5e7c905d37ba7e392083de1a249cc8a1.